apps/common-app/src/examples/LayoutAnimations/BasicLayoutAnimation.tsx

```diff
@@ -17,14 +13,23 @@ export default function BasicLayoutAnimation() {
       <Button onPress={() => setState(!state)} title="Update" />
       {visible && (
         <Animated.View
-          entering={BounceInLeft}
-          layout={Layout.springify()}
-          exiting={FadeOutRight.duration(500)}
+          layout={LinearStyleTransition.duration(1000)}
```

Comment:
i think it would be better to keep it as new props called for example `transition`

Reply:
I think it's a good idea, but I am not sure how would you like layoutTransition and styleTransition to overwrite each others props. Perhaps we keep it for now the way it is?